packages/terra-functional-testing/tests/wdio/terra-validates-spec.js

```diff
@@ -37,6 +37,15 @@ Terra.describeViewports('Terra.validates', ['small', 'large'], () => {
       Terra.validates.screenshot('validates-screenshot');
     });
 
+    it('should pass if ignoreScreenshotMistmatch is true even when mismatched', () => {
+      const oldIgnoreScreenshotMistmatch = Terra.serviceOptions.ignoreScreenshotMistmatch;
+      Terra.serviceOptions.ignoreScreenshotMistmatch = true;
+      browser.url('/insufficient-color-contrast.html');
+
+      Terra.validates.screenshot('validates-mismatch');
```

Comment:
I think this test is misleading and not testing what we think. There is no mismatch when comparing this screenshot and test will pass even when ignoreScreenshotMismatch was set to false because it should always match. I wonder if we really need this test since it is already being covered in jest and this is not generating a mismatch screenshot.

Reply:
I copied a different test image and saved it as the name for this test specifically to make sure there was a mismatch (couldn't find a way to test this without doing it that way). 

I do think I agree that we might not need it with jest, however we may want it for the next step of this series of changes, where whatever reporter we end up using/making reports that there are mismatches.

Reply:
Ah I see. However, that different test image used here will get deleted when ever the `updateScreenshots` option is used to regenerate all screenshots.

Reply:
Removed teh wdio test and screenshots here https://github.com/cerner/terra-toolkit/pull/655/commits/9b9e06d9786ebf0471e3f3e0f7c2df80639a004c#diff-c942bb45306f90f41bcaffbfcfecbcde32862b69523089e67cec126b30328451L40